src/include/homeobject/pg_manager.hpp

```diff
@@ -9,7 +9,8 @@
 
 namespace homeobject {
 
-ENUM(PGError, uint16_t, UNKNOWN = 1, INVALID_ARG, TIMEOUT, UNKNOWN_PG, UNKNOWN_PEER, UNSUPPORTED_OP);
+ENUM(PGError, uint16_t, UNKNOWN = 1, INVALID_ARG, TIMEOUT, UNKNOWN_PG, UNKNOWN_PEER, UNSUPPORTED_OP, PG_ALREADY_EXISTS,
```

Comment:
We should not introduce an error for something that is intended to be idempotent. If a PG exists; just return SUCCESS? There may be instances where the CM sends this to one SM; it times out, but actually works, then sends it again either to the same SM or one of the other members. Should just be instant success.

Reply:
make sense, will remove this error type